### ISSUE TITLE
lofi chill

### ISSUE THREAD

**Issue by Sugarbear60** [opera-gaming/gxmods#43]
the lofi chill background music seems to not be working.

**Comment by astoh3** [comment]
it works with me make sure to chek these following stuff:
- background music is not muted from the side bar mute button.

![20230228_174259 00_00_00-00_00_30](https://user-images.githubusercontent.com/125594340/221888827-040131a9-72cf-4044-ad05-6abefdbfd077.gif)

- background music is not turned off from the background music settings.

![20230228_174038 00_00_00-00_00_30](https://user-images.githubusercontent.com/125594340/221889162-bdaa2694-f4cc-4ca3-83fa-71b997422263.gif)

- make sure to increase the volume lvl to a any higher lvl than the current.

![20230228_174122 00_00_00-00_00_30](https://user-images.githubusercontent.com/125594340/221889314-e4672bc4-4411-4f60-9cc7-b49d0500e65c.gif)

- check if there is any mod is given the priority from the mods page, you can do it by placing the mod on top.

![20230228_174418 00_00_00-00_00_30](https://user-images.githubusercontent.com/125594340/221889664-d2fd7472-72ca-4287-8ee4-5d97949df6d5.gif)

- check if the background music if enabled from the mod its self.

![20230228_174516 00_00_00-00_00_30](https://user-images.githubusercontent.com/125594340/221890684-6ac8873e-2557-487b-8c9b-0159003c755d.gif)



if non of these worked then make a bug issue about it.

**Comment by astoh3** [comment]
> Nicely put. I will add to that one thing. Background music keeps playing only if browser doesn't play any music, so if there's Youtube playing music is automatically muted.

Yes i forgot to mention this out the background music will be automatically muted whenever there is "other sound source" is playing, like youtube, spotify, twitch. btw the background music will still play if the vid you are watching dose not nave any audio source.

and so thanks for @themaciekk for pointing this out 👍
